Dockerfile

```diff
@@ -70,7 +69,7 @@ FROM no-spark AS with-spark-optional-dag
 
 # Install Java
 RUN apt-get update \
-    && apt-get install -t jessie-backports --no-install-recommends -y openjdk-8-jre-headless \
+    && apt-get --no-install-recommends -y install default-jre \
```

Comment:
Hmm suggest to stick with the explicit `openjdk-8-jre` over `default-jre`. After all `JAVA_HOME` below points explicitly to Java 8, and Spark cannot use anything > Java 8.

I think it's also better to stick with `headless` since it is unlikely one uses this for GUI. See comparison: https://www.quora.com/What-is-the-difference-between-openjdk-jre-headless-and-openjdk-jre-jdk. If the user really needs the GUI, he/she can opt to install `openjdk-8-jre` on top of `openjdk-8-jre-headless` anyway.

Reply:
reverted